utils/model.py

```diff
@@ -104,11 +104,11 @@ def __init__(self, in_planes, kernel_size, stride, padding, n_basis_kernels, tem
         if hidden_planes < 4:
             hidden_planes = 4
 
-        if not pool_dim == 'both':
```

Comment:
this was a bug I think? I think this would change any value do to `(not pool_dim) == 'both'` => `False == 'both'`

Reply:
actually, nvm, just realized that I didn't fix it below and it still works fine (seems weird to me haha)

Reply:
yeah it works fine and it's kinda weird haha... I thought the same thing but found out it works find and didn't bother to correct it. Thanks for pointing it out anyway!